scripts/build/termux_step_get_dependencies.sh

```diff
@@ -70,15 +72,9 @@ termux_step_get_dependencies() {
 				(
 					cd $TERMUX_COMMON_CACHEDIR-$DEP_ARCH
 					if [ "$TERMUX_REPO_PKG_FORMAT" = "debian" ]; then
-						ar x ${PKG}_${DEP_VERSION}_${DEP_ARCH}.deb data.tar.xz
-						if tar -tf data.tar.xz|grep "^./$">/dev/null; then
-							# Strip prefixed ./, to avoid possible
-							# permission errors from tar
-							tar -xf data.tar.xz --strip-components=1 \
-								--no-overwrite-dir -C /
-						else
-							tar -xf data.tar.xz --no-overwrite-dir -C /
-						fi
+						# Ignore topdir `.`, to avoid possible  permission errors from tar
+						ar p ${PKG}_${DEP_VERSION}_${DEP_ARCH}.deb data.tar.xz | \
+							tar xJ --no-overwrite-dir --transform='s#^.$#data#' -C /
```

Comment:
`tar` has a god damn substitution option???

Reply:
Yep. I also tried `--exclude=.` but it excludes the whole folder. Permission error is relevant only to `/` folder so I fixed it this way.

Reply:
I think I can mark it as resolved.